libbeat/docs/upgrading.asciidoc

```diff
@@ -348,3 +367,44 @@ You might be writing to an index that doesn't match the index pattern used
 by {beats}. See <<non-es-outputs>>.
 
 
+[[dashboard-shard-failed]]
+==== "shard failed" error when viewing {beats} dashboards in {kib}
+
+After upgrading to {es} 7.0, any indices created by Beats 6.6 or older will not
+work in {kib} dashboards until the `index.query.default_field` setting is added
+to each index. Indices created in Beats 6.7 or later are unaffected.
+
+*Background:* Starting in {es} 7.0, some query types, such as Simple Query
+String, have a limit to the number of fields they will query against. Because
+Beats indices often contain more fields than this cap, the
+`index.query.default_field` index setting is used to inform {es} which fields to
+use by default when no field is specified for a query.
+
+To add the setting to the index, you can use the 7.0
```

Comment:
@joshdover I don't mention license requirements here because the docs about the upgrade assistant cover that.